module_text_llm/module_text_llm/suggest_feedback_basic.py

```diff
@@ -48,8 +48,8 @@ async def suggest_feedback_basic(exercise: Exercise, submission: Submission, con
             continue
 
         try:
-            credits = float(row["credits"])
-        except ValueError:
+            feedback_credits = float(row["credits"])
+        except (ValueError, TypeError):
             logger.warning("Could not parse credits from row %s", row)
```

Comment:
```suggestion
            logger.warning("Could not parse credits from row %s", row)
            traceback.print_exc()
```
to see the stack trace for debugging

Reply:
I completely refactored it in #79, but will add it anyways 👍